packages/models-library/src/models_library/docker.py

```diff
@@ -1,8 +1,11 @@
 import re
-from typing import Optional
+from typing import Final, Optional
 
 from pydantic import ConstrainedStr, constr
 
+_DOCKER_LABEL_KEY_REGEX: Final[re.Pattern] = re.compile(
+    r"^(?!(\.|\-|com.docker\.|io.docker\.|org.dockerproject\.))(?!.*(--|\.\.))[a-z0-9\.-]+$"
```

Comment:
can you please add a test to check this PR? 
It is very handy to have one at hand in case we find a corner case 
You can add one very fast using ``packages/models-library/tests/test_basic_regex.py``

Reply:
of course